javascript-loops/main.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable prefer-const */
```

Comment:
Don't add any lint rules without asking me please. Lint errors are there for a reason, so if you need to add a rule to get past an error, you're probably doing something incorrect.